webclient/app/src/app/commons/entityCard/EntityCard.jsx

```diff
@@ -24,39 +24,43 @@ function EntityCard(props) {
 
     function renderAttributes(entity) {
 
-        return entity.fields.map((field, index) => {
+        if (!entity.fields || entity.fields.length === 0) {
             return (
-
-                <TableRow key={index}>
-                    <TableCell className={entityCardStyles.relationshipPlaceholder}>
-                        <div className={"anchor_" + entity.name + "_" + field.fieldName + "_l"}/>
-                    </TableCell>
-                    <TableCell>{field.fieldName}</TableCell>
-                    <TableCell>{field.fieldType}</TableCell>
-                    <TableCell className={entityCardStyles.relationshipPlaceholder}>
-                        <div className={"anchor_" + entity.name + "_" + field.fieldName + "_r"}/>
+                <TableRow key={0}>
+                    <TableCell colSpan={4} >
+                        <div className={entityCardStyles.statementWrapper}>
```

Comment:
I do not know, this is best practice to place this inside of an table

Reply:
I need a places to add the relationship anchors even if there is no field defined